static/styles/portico.scss

```diff
@@ -2029,13 +2005,6 @@ input.new-organization-button {
-    font-weight: normal;
-    font-style: normal;
-}
```

Comment:
We don't seem to be using the `Stamp` font anywhere in the codebase anymore, so I just removed it thinking it was extraneous code. Let me know if we still do.

Reply:
This sort of "change you noticed we should make while doing something" belongs in its own commit with basically what you just wrote in your comment as the commit message; doing that makes it a lot easier for reviewers (since they don't need to guess why you removed it and can quickly verify it's unused and merged).  I ended up looking up the history of this and merging 1381603c29f6bf9d815eb2e8d60b86a2315004b1 as a commit to remove this block.  

Anyway, thanks for catching this!  This font was about 40% of the total size of portico.scss :)